packages/design-system/src/components/web-components/preactement/__snapshots__/define.test.tsx.snap

```diff
@@ -0,0 +1,20 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`define() when run in the browser renders child HTML 1`] = `
+<description-pair
+  term="supercalifragilisticexpialidocious"
```

Comment:
🤣

Reply:
I really struggled to come up with an example component that would test the right things in my tests (needed two sibling components under no common wrapper element), and then what better word to define than this? 😂